src/public/version.h

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NINJA_PUBLIC_VERSION_H_
+#define NINJA_PUBLIC_VERSION_H_
+
+namespace ninja {
```

Comment:
I don't have a problem with putting the version symbol in the namespace ninja, but you might get a minor pushback from doing this "early" in the patch series, since putting symbols inside the ninja namespace appears to be its own commit later on.

Reply:
Ooops, got my commits out of order in my head. Never mind this.